src/containers/Auth/Login.js

```diff
@@ -41,13 +48,11 @@ const Login = ({ history, loginUser, clearError }) => {
   }
   return (
     <div className='container'>
-      <div className='row loginRow'>
-        <div className='col-sm-5 offset-sm-4 loginColumn'>
+      <LoginRow className='row'>
+        <LoginColumn className='col-sm-5 offset-sm-4'>
           <form>
```

Comment:
Can we use StyledForm here, with styled-components, we usually don't use the raw html tags in the jsx.